### PR TITLE
Dockerfile fixes

### PR DESCRIPTION
**Pull request by gbicann**
This merge request fixes the Docker entrypoint by avoiding hardcoding the version number, instead it's extracted from `pom.xml`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest
 
-RUN apk -q update ; apk -q add maven git jo
+RUN apk -q update ; apk -q add maven git xq
```
Comment (sraveh) [inline Dockerfile:3]: why are we building the app inside docker. This should be change and move to build time

**Comment by sraveh** [comment]
@gbicann @anewton1998 we need to discuss how the application being built and package. This does not follow our best practices

File: Dockerfile (hunk 1 above)
Comment (anewton1998) [reply Dockerfile:3]: Not everybody has Java and Maven installed on their systems, so this allows them to build and run the conformance tool.

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -19,8 +19,12 @@ for arg in "$@" ; do
   fi
 done
 
-java -jar tool/target/rdapct-1.0.4.jar -c tool/bin/rdapct_config.json --use-local-datasets "$@" 1>&2
+RCT_VERSION="$(xq -x //rdap-conformance.version pom.xml)"
```
Comment (anewton1998) [inline docker-entrypoint.sh:22]: As an FYI, this version.properties file will also have the version number in it. But this works too. :)

**Pull request merged**